node/task.ts

```diff
@@ -539,28 +539,22 @@ export interface FsStats extends fs.Stats {
 }
 
 /**
- * Get's stat on a path. 
+ * Get's stat on a path.
  * Useful for checking whether a file or directory.  Also getting created, modified and accessed time.
  * see [fs.stat](https://nodejs.org/api/fs.html#fs_class_fs_stats)
- * 
+ *
  * @param     path      path to check
- * @returns   fsStat 
+ * @returns   fsStat
  */
 export function stats(path: string): FsStats {
     return fs.statSync(path);
 }
 
 export const exist = im._exist;
 
-export interface FsOptions {
-    encoding?: string;
-    mode?: number;
-    flag?: string;
-}
-
-export function writeFile(file: string, data: string | Buffer, options?: string | FsOptions) {
-    if(typeof(options) === 'string'){
-        fs.writeFileSync(file, data, {encoding: options});
+export function writeFile(file: string, data: string | Buffer, options?: BufferEncoding | fs.WriteFileOptions) {
+    if (typeof(options) === 'string'){
```

Comment:
With the updated input types, is it ever possible for this to return true?

Reply:
`BufferEncoding` is a type that is a list of possible strings. typeof() should still resolve to `string`